oscplot.c

```diff
@@ -3039,11 +3039,13 @@ static void channels_transform_assignment(GtkTreeModel *model,
 			prm->ch_settings = g_slist_reverse(prm->ch_settings);
 			transform = add_transform_to_list(plot, CROSS_CORRELATION_TRANSFORM, prm->ch_settings);
 		}
+		break;
```

Comment:
This looks more like a bug fix!

Reply:
created PR #80 for this
will rebase on top of it, once merged;